DB.java

```diff
@@ -294,6 +294,85 @@ else if(i==0)
 		
 		return result;
 	}
+	
+	public boolean check_comments_popular(String ip, int id, int a) throws SQLException

```

Comment:
@mjhmohamed 

Need to comment to describe the purpose of this method.

Hui

Reply:
unfortunately, our programmer seems to be unavailable for some reasons.
So, I'll try my best to comment it as far as my understanding.

IBRAHIM.